src/runtime/crt/crt_config.h.template

```diff
@@ -17,47 +17,53 @@
  * under the License.
  */
 
-/*!
- * \file tvm/runtime/crt_config.h.template
- * \brief Template for CRT configuration, to be modified on each target.
+ /*!
```

Comment:
nit: something happened with spacing and slashes here

Reply:
fixed this